src/components/UI/atoms/Link/Link.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { fontSizes } from '<variables>/fonts';
+
+/**
+ * @description - Link Component
+ *
+ * @prop {boolean} isTag
+ * @prop {string} fontSize
+ * @prop {string} href
+ * @prop {children} children - children
+ *
+ * @return {component} Link
+ */
+const Link = ({
+  href,
+  isTag,
+  fontSize,
+  margin,
+  onClick,
+  children,
+}) => (
+  <Link.Container
+    href={href || '#'}
```

Comment:
Don't use href use Link from `react-router-dom` so you have access to the apps internal routes. You might need to mock Router for the tests as I don't think you are allowed to render links in isolation